src/SendEmail.php

```diff
@@ -24,6 +24,10 @@ private function postRequest(): Response
     {
         $client = new Client();
 
+        if (!($this->data['data'] && count($this->data['data']))) {
+            $this->data['data'] = (object)[];
+        }
+
```

Comment:
Isn't this going to fail on the validate() function? if data is empty

Reply:
its a little confusing, but validate checks this->data, this is checking this->data->data set on buildEmailContent. I also tested with web app both empty and with an array and both went to sales force.